src/examples/Blending.cpp

```diff
@@ -132,7 +132,7 @@ void tvgDrawCmds(tvg::Canvas* canvas)
 
     //Lighten
     auto picture = tvg::Picture::gen();
-    if (picture->load(data, 200, 300, true) != tvg::Result::Success) return;
+    if (picture->load(data, 200, 300, false, true) != tvg::Result::Success) return;
```

Comment:
Exisiting default value was true.

Reply:
it keep true (the lastone parameter)

Reply:
The last one indicates "copy", and we used the premultiplied version before. You don't need to break it.
See: tvgRawLoader.cpp:92

Reply:
Git it. Fixed